### PR TITLE
RHOAIENG-9788: Create a sample S3 fetch pipeline in manifests

### PR DESCRIPTION
**Pull request by jackdelahunt**
## Description
Adds a new "s3-fetch" pipeline to the manifests directory. E2E tests now use this new pipeline and so does the shell tests

## How Has This Been Tested?
- Ran e2e test locally and passed
- Ran shell tests locally and passed
- Ran shell tests in CI and passed

## Merge criteria:

- [x] The commits are squashed in a cohesive manner and have meaningful messages.
- [x] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [x] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

**Comment by LaVLaS** [comment]
Looks good. The only change we should make is to included a reference to the model name in the pipelineRun file name: `s3-fetch.bike-rentals.pipelinerun.yaml`

File: test/e2e-tests/tests/pipelines_test.go [hunk 1]
```diff
@@ -11,7 +11,7 @@ import (
 )
 
 const (
-	AIEdgeE2EBikeRentalsPipelineRunRelativePath       = support.AIEdgeE2EDirectoryRelativePath + "/aiedge-e2e.bike-rentals.pipelinerun.yaml"
+	S3FetchBikeRentalsPipelineRunRelativePath         = support.AIEdgeE2EDirectoryRelativePath + "/s3-fetch.pipelinerun.yaml"
```
Comment (LaVLaS) [inline test/e2e-tests/tests/pipelines_test.go:14]: The filename needs to be updated here also

```suggestion
	S3FetchBikeRentalsPipelineRunRelativePath         = support.AIEdgeE2EDirectoryRelativePath + "/s3-fetch.bike-rentals.pipelinerun.yaml"
```

Comment (jackdelahunt) [reply test/e2e-tests/tests/pipelines_test.go]: Done

**Comment by LaVLaS** [comment]
/approve
/lgtm


Looks great

**Pull request merged**